WcaOnRails/app/models/registration.rb

```diff
@@ -18,6 +18,7 @@ class Registration < ApplicationRecord
   has_many :events, through: :competition_events
   has_many :assignments, dependent: :delete_all
   has_many :wcif_extensions, as: :extendable, dependent: :delete_all
+  has_many :stripe_payment_intents, as: :holder, dependent: :delete_all
```

Comment:
Having `registration_payments`, `stripe_payment_intents`, and `stripe_transactions` seems like similar ideas. What do each of these do? Is there a way we could remove one/derive it instead?

Reply:
✅ Discussed on Google Meet: Separation of `registration_payments` from `stripe_payment_intents` allows for going the multi-processor route later. `stripe_transactions` serves as additional logging.